internal/v2/repository/iam/aggregate.go

```diff
@@ -2,6 +2,18 @@ package iam
 
 import (
 	"context"
+	"github.com/caos/zitadel/internal/v2/repository/iam/policy/label"
+	iam_login "github.com/caos/zitadel/internal/v2/repository/iam/policy/login"
+	"github.com/caos/zitadel/internal/v2/repository/iam/policy/login/idpprovider"
+	iam_multi_factor "github.com/caos/zitadel/internal/v2/repository/iam/policy/login/multi_factors"
+	iam_second_factor "github.com/caos/zitadel/internal/v2/repository/iam/policy/login/second_factors"
```

Comment:
analog https://blog.golang.org/package-names bad package names. maybe a package `factors` with both factor types in it would be better